frontend/src/metabase/admin/settings/selectors/selectors.js

```diff
@@ -401,15 +401,32 @@ export const ADMIN_SETTINGS_SECTIONS = {
     settings: [],
     isUpsell: true,
   },
-  whitelabel: {
+  appearance: {
+    // OSS Version
     name: t`Appearance`,
-    getHidden: (settings) => settings["token-features"]?.whitelabel === true,
+    getHidden: (settings) => settings["token-features"]?.whitelabel,
     order: 133,
-    component: (props) => (
-      <UpsellWhitelabel {...props} source="settings-appearance" />
-    ),
-    settings: [],
+    component: () => <AppearanceSettingsPage />,
     isUpsell: true,
+    settings: [],
+  },
+  whitelabel: {
+    // EE Version
+    name: t`Appearance`,
+    getHidden: (settings) => !settings["token-features"]?.whitelabel,
+    order: 134,
+    component: () => <AppearanceSettingsPage />,
+    settings: [],
+  },
+  "whitelabel/branding": {
+    name: t`Appearance`,
+    component: () => <AppearanceSettingsPage />,
+    settings: [],
+  },
+  "whitelabel/conceal-metabase": {
+    name: t`Appearance`,
+    component: () => <AppearanceSettingsPage />,
+    settings: [],
   },
```

Comment:
this is a silly bit of routing nonsense, but necessary until we can get rid of this data-based-router thingy